tests/tools/lib/replay_schema.json

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
Can we put AL2 header here?

Reply:
What is an AL2 header?

Reply:
Apache License 2

Reply:
On it!

Reply:
I think the best option is to add to the `description` field the text "Licensed under Apache V2 https://www.apache.org/licenses/LICENSE-2.0".